### ISSUE TITLE
Make complain validate if there's two <releases> tags

### ISSUE THREAD

**Issue by tsdgeos** [hughsie/appstream-glib#456]
Would be nice if it complained if the appdata file had stuff like

<releases>
   <release version="22.04.3" date="2022-07-07"/>
   <release version="22.04.2" date="2022-06-09"/>
   <release version="22.04.1" date="2022-05-12"/>
   <release version="22.04.0" date="2022-04-21"/>
 </releases>
 <releases>
   <release version="3.1.0" date="2019-03-19"/>
 </releases>

**Comment by hughsie** [comment]
Makes sense, but at this point it probably makes sense to add this check to https://github.com/ximion/appstream instead.

**Comment by ximion** [comment]
AFAIR `appstreamcli` was testing for this from the start, so this should already be covered :-)
